lib/cli/query.ts

```diff
@@ -141,6 +141,8 @@ function argvQueryPrefold(argv, implicitTimestampOps) {
   if (argv["timestamp-attribute"] && argv["timestamp-attribute"].length > 0)
     ts_attr = argv["timestamp-attribute"];
 
+  // TODO(cstrahan): actually reverse the results?
+  let reverse = 1;
```

Comment:
Bug fix: the line that sets reverse below was undeclared. Also, there's nowhere that we end up using this, so we should probably actually implement the reversing logic.